ls1intum/Artemis
# [Feature] Collapsable list item for Feedback

### Checklist

### Motivation and Context
 Feedback items have long text making the feedback view crowded
 #3217 

### Description
 added angular/material and angular/cdk to be able to use https://material.angular.io/components/expansion/overview 
which i then used to create feedback-collapse component to make feedback un-/collapsable

### Steps for Testing

0. Be logged in as instructor
1. Navigate: Course Management -> Some Course -> Exercises -> If there are none generate a basic one for testing -> Edit in Editor 
2. Now you should be able to see the feedback on the top right
3. if none of the feedback messages are over 200 characters go in the test repository and change a message to be longer 
![testing_feedback](https://user-images.githubusercontent.com/56715566/115598168-653d0480-a2da-11eb-91b4-5e9397410801.gif)
4. Save and submit
5. look at feedback


### Test Coverage



TODO

### Screenshots


![feedback-collapse-final](https://user-images.githubusercontent.com/56715566/115754860-f07ece80-a39c-11eb-9574-54d52ab87784.gif)
